### DISCUSSION TITLE
Logout and NavigationViewCoordinator using

### DISCUSSION THREAD

**Discussion by 2jumper3** [rundfunk47/stinsen#114] [Q&A]
Hi all! 
Please can somebody explain and help me. 
I'm trying to push TabView controller after login, and if I understand clearly Stinsen, I can do it with 2 ways: 
    _func makeTabView() -> NavigationViewCoordinator<TabBarCoordinator> {
        return NavigationViewCoordinator(TabBarCoordinator())
    }_
and
    _@ViewBuilder func makeAuthenticated() -> some View {
        TabBarController()
    }_
But I cant understand, which is better? If I use first, every tabs on my TabView getting new Navigationbar and I cant hide it. 
<img width="373" alt="Снимок экрана 2022-12-31 в 17 19 52" src="https://user-images.githubusercontent.com/45697468/210138119-78cd6ed8-3d54-4b4c-92a0-57a3e6c942e4.png">

When I should use first method and when second? 
By the way, I cant make logout from 1 tab. My logout method make login screen only on tab where I calling it. Tab view still on the screen and cant understand how to remove it if I writing _@Root var login = makeLogin_
<img width="349" alt="Снимок экрана 2022-12-31 в 17 19 17" src="https://user-images.githubusercontent.com/45697468/210138110-92f1da40-86e8-42a5-9460-eed2c3502290.png">

**Comment by 2jumper3** [comment]
Still actual...)

**Comment by rundfunk47** [comment]
Hi! Wrapping something in a NavigationViewCoordinator is the same as wrapping a view in NavigationView using vanilla SwiftUI. If you don't want the Navigation bar, simply don't use the NavigationViewController :)

**Reply by 2jumper3** [reply]
Hi @rundfunk47  and many thanks for your reply! 
I've got it! Thanks! 
Maybe you know something about another problem: 
If I use `customise` on TabBarCoordinator like this: 
<img width="495" alt="Снимок экрана 2023-02-10 в 08 46 55" src="https://user-images.githubusercontent.com/45697468/218003173-72ae691d-8fe2-4867-bf60-fe1cc062a72d.png">

I cant hide this overlayed views sometimes. If should use `customise` method because I need another design... but this is what happened: 

https://user-images.githubusercontent.com/45697468/218003443-b4f08bf1-85d8-40dc-ac25-030d6c7ede0a.mov

I'm using this method from example `App`, and can't understand how to hide other images.. 
<img width="711" alt="Снимок экрана 2023-02-10 в 08 50 13" src="https://user-images.githubusercontent.com/45697468/218003568-8d414fd5-5a75-4ab1-9934-1eee49884050.png">

Many thanks, and have a good day.
